src/ui/components/OrderList/components/OrderListItem.tsx

```diff
@@ -0,0 +1,147 @@
+import Image from "next/image";
+import Link from "next/link";
+import { PaymentStatus } from "./PaymentStatus";
+import { formatDate } from "@/lib/date";
+import { formatMoney } from "@/lib/graphql";
+import { getHrefForVariant } from "@/utils/getHrefForVariant";
+import { type OrderDetailsFragment } from "@/gql/graphql";
+
+type Props = {
+	order: OrderDetailsFragment;
+};
+
+export const OrderListItem = ({ order }: Props) => {
+	return (
+		<li className="bg-white">
+			<div className="flex flex-col gap-2 border bg-neutral-200/20 px-6 py-4 md:grid md:grid-cols-4 md:gap-8">
+				<dl className="flex flex-col divide-y divide-neutral-200 text-sm md:col-span-3 md:grid md:grid-cols-3 md:gap-6 md:divide-none lg:col-span-2">
+					<div className="flex flex-row items-center justify-between py-4 md:flex-col md:items-start md:gap-y-1">
+						<dt className="font-medium text-neutral-900">Order number</dt>
+						<dd className="text-neutral-600">{order.number}</dd>
+					</div>
+					<div className="flex flex-row items-center justify-between py-4 md:flex-col md:items-start md:gap-y-1">
+						<dt className="font-medium text-neutral-900">Date placed</dt>
+						<dd className="text-neutral-600">
+							<time dateTime={order.created}>{formatDate(new Date(order.created))}</time>
+						</dd>
+					</div>
+					<div className="flex flex-row items-center justify-between py-4 md:flex-col md:items-start md:gap-y-1">
+						<dt className="font-medium text-neutral-900">Payment status</dt>
+						<dd>
+							<PaymentStatus status={order.paymentStatus} />
+						</dd>
+					</div>
+				</dl>
+				{/* TODO: Reveal after implementing the order details page. */}
+				{/* <div className="flex flex-col md:col-span-1 md:flex-row md:items-center lg:col-span-2">
+					<Link
+						href="#"
+						className="flex items-center justify-center rounded border border-neutral-200 bg-white px-4 py-2 text-sm hover:bg-neutral-50 focus:bg-neutral-50 md:ml-auto"
+					>
+						View Order
+					</Link>
+				</div> */}
+			</div>
+
+			{order.lines.length > 0 && (
+				<>
+					<div className="md:border-x md:px-6">
+						<table className="w-full text-sm text-neutral-500">
+							<thead className="sr-only">
+								<tr>
+									<td>product</td>
+									<td className="max-md:hidden">quantity and unit price</td>
+									<td>price</td>
+								</tr>
+							</thead>
+							<tbody className="md:divide-y">
+								{order.lines.map((item) => {
+									if (item.variant) {
```

Comment:
Reverse the condition;
```suggestion
									if (!item.variant) {
										return null;
									}
```